### ISSUE TITLE
[Bug?] Starting a built project trigger error: `Cannot find module '.../.output/server/node_modules/.../index.js' imported from .../.output/server/index.mjs`

### ISSUE THREAD

**Issue by lominming** [solidjs/solid-start#1296]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Running into a new issue since upgrading to `0.4.0` (beta2)

build works `vinxi build` but starting the built project `vinxi start` will fail with this error:

`Cannot find module '.../.output/server/node_modules/drizzle-orm/postgres-js/index.js' imported from .../.output/server/index.mjs`

---

File `.output/server/index.mjs` looks like:
```
...
import 'drizzle-orm/postgres-js';
import 'drizzle-orm/postgres-js/migrator';
import 'drizzle-orm';
...
```

And inside `.output/server/node_modules/drizzle-orm` is missing the folder `postgres-js`

### Steps to reproduce 🕹

Steps:

1.
2.
3.
4.

### Context 🔦

Since my built project cannot be started, I cannot actually deploy to production right now. 

### Your environment 🌎

```shell
Node: 20.5.1

"@solidjs/start": "^0.4.10"
"solid-js": "^1.8.12"
"vinxi": "^0.1.4"
```

**Comment by katywings** [comment]
This error is happening for me with a fresh `basic` example project, just from executing `pnpm build && pnpm start`

**Comment by ryansolid** [comment]
Hmm.. odd.. I just did that like 10 mins ago in the vercel repo and had no issue. How can we have a drizzle orm reference error in the basic example?

**Comment by katywings** [comment]
Sorry yeah, my error is slighly different: `Cannot find module '/home/geronimo/Projects/solid-start-b2/anotherone/.output/server/node_modules/solid-js/web/dist/server.js' imported from /home/geronimo/Projects/solid-start-b2/anotherone/.output/server/index.mjs`

**Comment by katywings** [comment]
@lominming In my case the parent folder `Projects` had a package.json and node_modules, which interfered with the nested project. Removing the `package.json` and `node_modules` from the parent folder, reinstalling the dependencies in my project and rebuilding, fixed my issue.

**Comment by joeygrable94** [comment]
I am getting a similar error in my project but the exception is being caused byt the dom-helpers/activeElement dependency. Could this be the same issue?

———

PROCESS app % node .output/server/index.mjs
node:internal/process/esm_loader:34

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/project/frontend/app/.output/server/node_modules/dom-helpers/activeElement' imported from /project/frontend/app/.output/server/index.mjs

Did you mean to import "dom-helpers/cjs/activeElement.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:263:11)
    at moduleResolve (node:internal/modules/esm/resolve:908:10)
    at defaultResolve (node:internal/modules/esm/resolve:1131:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:390:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:359:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:234:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///project/frontend/app/.output/server/node_modules/dom-helpers/activeElement'
}

**Comment by ryansolid** [comment]
Without a reproduction we are just attracting other bundling issues. I'm going to close this as stale.

**Comment by andreiborza** [comment]
I've encountered this in a monorepo (using Nx) with pnpm. I ended up getting two versions of solid-js in the `.output` folder, `1.8.18` and `1.8.17`. The `1.8.18` version lacked the `web` folder which is where this issue comes from.

It works outside of our monorepo, with pnpm. I also wasn't able to reproduce a breaking monorepo in isolation. Just thought this might help further narrow it down anyway.
